coordinator/src/main/scala/filodb.coordinator/queryengine2/QueryEngine.scala

```diff
@@ -133,24 +133,23 @@ class QueryEngine(dataset: Dataset,
                                   submitTime: Long,
                                   options: QueryOptions, spreadProvider: SpreadProvider): PlanResult = {
     logicalPlan match {
-      case lp: RawSeries => materializeRawSeries(queryId, submitTime, options, lp, spreadProvider)
-      case lp: RawChunkMeta => materializeRawChunkMeta(queryId, submitTime, options, lp,
-        spreadProvider)
-      case lp: PeriodicSeries => materializePeriodicSeries(queryId, submitTime, options, lp,
-        spreadProvider)
+      case lp: RawSeries =>                   materializeRawSeries(queryId, submitTime, options, lp, spreadProvider)
+      case lp: RawChunkMeta =>                materializeRawChunkMeta(queryId, submitTime, options, lp, spreadProvider)
+      case lp: PeriodicSeries =>              materializePeriodicSeries(queryId, submitTime, options, lp,
+                                              spreadProvider)
       case lp: PeriodicSeriesWithWindowing => materializePeriodicSeriesWithWindowing(queryId, submitTime, options, lp,
-        spreadProvider)
-      case lp: ApplyInstantFunction => materializeApplyInstantFunction(queryId, submitTime, options, lp,
-        spreadProvider)
-      case lp: Aggregate => materializeAggregate(queryId, submitTime, options, lp, spreadProvider)
-      case lp: BinaryJoin => materializeBinaryJoin(queryId, submitTime, options, lp, spreadProvider)
+                                              spreadProvider)
+      case lp: ApplyInstantFunction =>        materializeApplyInstantFunction(queryId, submitTime, options, lp,
+                                              spreadProvider)
+      case lp: Aggregate =>                   materializeAggregate(queryId, submitTime, options, lp, spreadProvider)
+      case lp: BinaryJoin =>                  materializeBinaryJoin(queryId, submitTime, options, lp, spreadProvider)
       case lp: ScalarVectorBinaryOperation => materializeScalarVectorBinOp(queryId, submitTime, options, lp,
-        spreadProvider)
-      case lp: LabelValues => materializeLabelValues(queryId, submitTime, options, lp, spreadProvider)
-      case lp: SeriesKeysByFilters => materializeSeriesKeysByFilters(queryId, submitTime, options, lp,
-        spreadProvider)
-      case lp: ApplyMiscellaneousFunction => materializeApplyMiscellaneousFunction(queryId, submitTime, options, lp,
-        spreadProvider)
+                                              spreadProvider)
```

Comment:
Sorry minor nit, could you also have the => line up as well?  Thanks  :)

Reply:
Did space adjustment